### ISSUE TITLE
Adapta GTK Theme causes screen tearing

### ISSUE THREAD

**Issue by drakofrost** [adapta-project/adapta-gtk-theme#602]
I installed Adapta GTK Theme on Fedora, and noticed that it causes screen tearing in Gnome MPV, the same doesn't happen with other themes. I installed the version available in the repositories.

 ```
 * Distribution - Fedora 27
 * Gtk+ 3.0 version -  3.22
 * Desktop environment - Gnome
 * Display server protocol - X11
 ```

**Adapta version (if you knew)**

3.92.2.63

**Related Application and/or Shell theming (or name of widget)**

Gnome MPV

**Actual issue**

Screen tearing.

**Steps to reproduce (if you knew)**

Play a video with lots of motion on Gnome MPV, tearing will appear in the top portion of the screen.

**Expected behaviour**

Screen tearing shouldn't happen.

**Comment by tista500** [comment]
I don't see any remarkable differences between Adwaita and Adapta at least in VA-API and VDPAU accel backends.

**Comment by tista500** [comment]
Any sample online movies and reproducible gnome-mpv settings?

**Comment by SolarLiner** [comment]
Cannot reproduce here either, with 3.93: that being said I'm using a Maia variant. I'll test again with upstream Adapta.

**Comment by drakofrost** [comment]
The package was updated to version 3.93.0.43-1 in Fedora repositories, and the issue is still present.

My Gnome MPV settings are the following:

fs=yes
vo=opengl
profile=opengl-hq
sub-font-size=40

I tested with the episode 122 of Dragon Ball Super, the tearing shows up right in the beginning. I will not post any links because they aren't legal.

I also found another issue, if I try to take a screenshot of Dota 2, using the Print Screen key, Gnome's top bar shows up in the image, as a transparency. I can't reproduce neither of those issues while using the Arc theme.

Since those issues are deal breakers to me, I uninstalled Adapta, therefore I will no longer be able to provide feedback.
